egui_demo_lib/src/apps/demo/tests.rs

```diff
@@ -205,47 +213,62 @@ impl super::Demo for TableTest {
     }
 }
 
-impl super::View for TableTest {
+impl super::View for GridTest {
     fn ui(&mut self, ui: &mut egui::Ui) {
         ui.add(
             egui::Slider::new(&mut self.min_col_width, 0.0..=400.0).text("Minimum column width"),
         );
         ui.add(
             egui::Slider::new(&mut self.max_col_width, 0.0..=400.0).text("Maximum column width"),
         );
-        ui.add(egui::Slider::new(&mut self.num_cols, 0..=5).text("Columns"));
+        ui.add(egui::Slider::new(&mut self.num_cols, 0..=8).text("Columns"));
         ui.add(egui::Slider::new(&mut self.num_rows, 0..=20).text("Rows"));
 
+        ui.horizontal(|ui| {
+            ui.add(egui::Checkbox::new(&mut self.striped, "Striped"));
```

Comment:
Maybe we should rename this to "highlight_rows` or something, and add a similar helper for columns?

"Striped" is ambiguous in the direction (my bad).